shared/chat/conversation/list.desktop.js

```diff
@@ -244,22 +248,56 @@ class ConversationList extends Component<void, Props, State> {
             onDeleteMessage={this.props.onDeleteMessage}
             onDownloadAttachment={() => { messageID && this.props.onLoadAttachment(messageID, filename) }}
             onOpenInFileUI={() => { downloadedPath && this.props.onOpenInFileUI(downloadedPath) }}
-            onHidden={() => this._hidePopup()}
+            onHidden={this._hidePopup}
             style={style}
           />
         )
     }
   }
 
+  _showEditor = (message: TextMessage, messageRect: any) => {
+    const popupComponent = (
+      <EditPopup
+        messageRect={messageRect}
+        onClose={this._hidePopup}
+        message={message.message.stringValue()}
+        onSubmit={text => { this.props.onEditMessage(message, text) }}
+        rect={messageRect}
+      />
+    )
+
+    // Have to do this cause it's triggered from a popup that we're reusing else we'll get unmounted
+    setImmediate(() => {
+      const container = document.getElementById('popupContainer')
+      // FIXME: this is the right way to render portals retaining context for now, though it will change in the future.
+      ReactDOM.unstable_renderSubtreeIntoContainer(this, popupComponent, container)
+    })
+  }
+
+  _findMessageFromDOMNode (start: any) : any {
```

Comment:
we get a click on the action thing, not the message holder itself so this finds it